components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/GeckoLoginStorageDelegate.kt

```diff
@@ -84,42 +84,3 @@ class GeckoLoginStorageDelegate(
         }
     }
 }
-
-/**
- * Converts an Android Components [Login] to an Application Services [ServerPassword]
- */
-fun Login.toServerPassword() = ServerPassword(
-    // Underlying Rust code will generate a new GUID if `guid` is missing.
-    id = guid ?: "",
-    username = username,
-    password = password,
-    hostname = origin,
-    formSubmitURL = formActionOrigin,
-    httpRealm = httpRealm,
-    timesUsed = timesUsed,
-    timeCreated = timeCreated,
-    timeLastUsed = timeLastUsed,
-    timePasswordChanged = timePasswordChanged,
-    // usernameField & passwordField are allowed to be empty when
```

Comment:
@mhammond this is where I plucked those usernameField for to keep consistency with what's expected here. I should still add the comments to the new ones though